### PR TITLE
Move submenu js to webserve

### PR DESCRIPTION
**Pull request by OmgImAlexis**
@miigotu @MGaetan89 When we can, more of the javascript needs to be moved to files instead of inline and added to the inc_bottom.mako instead of the top as it'll increase performance by loading the files after the page is done rendering. This won't change any functionality.

### CODE REVIEW

**Comment by WebSpider** [comment]
Does this relate to the issue that was reported that in some browsers the layout partially loads, and only after time it completes?

I'm in the car now, cant look up the #

-----Original Message-----

**Comment by OmgImAlexis** [comment]
Yep, this should fix it. I still want to move more of the js to the bottom though but I can't do that until we get rid of all of the inline js as most of it calls scripts and that'd end up calling the scripts before they're loaded.

**Comment by MGaetan89** [comment]
:+1: I'm not even sure we need JS to do that. It looks wrong to me. But it would need some big changes to update that.

**Comment by WebSpider** [comment]
Its SiCKRAGETV/sickrage-issues#2381

**Comment by miigotu** [comment]
The js needs moved to a separate .js file, and the browser will fix it.

**Comment by OmgImAlexis** [comment]
I've moved most of the icons to the webserve function thanks to @miigotu. The only ones that're left are ones that don't use the `ui-icon` class and the logout one that I can't find in webserve.

File: gui/slick/interfaces/default/inc_bottom.mako [hunk 1]
```diff
@@ -66,11 +66,15 @@
         <!--
```
Comment (OmgImAlexis) [inline gui/slick/interfaces/default/inc_bottom.mako]: Does anyone have any idea why this commented out section is in here?

Comment (MGaetan89) [inline gui/slick/interfaces/default/inc_bottom.mako]: I wondered that too.
I guess these options were here before being moved in the top menu.
I'm not sure if it is necessary to keep it.

**Comment by MGaetan89** [comment]
:+1: This PR is great :D

File: gui/slick/interfaces/default/inc_bottom.mako (hunk 1 above)
Comment (OmgImAlexis) [inline gui/slick/interfaces/default/inc_bottom.mako]: If no one can find any reason for it, that's more code we can remove. 

@WebSpider @neoatomic @fernandog @miigotu

Comment (miigotu) [inline gui/slick/interfaces/default/inc_bottom.mako]: I think it just used to be part of the submenu, before it was added to the navbar.

**Pull request merged**